api/pkg/block/volumelist.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 The OpenSDS Authors.
```

Comment:
List can be part of block. Business logic can be moved to respective folders i.e. block?

Reply:
Didn't get. The business logic is already part of block

Reply:
This is okay, I am saying about the refactoring of the code.
IMHO Since the directory is for API related logic services API api/pkg/block


I see a scope of code refactoring.

Reply:
Ok. We will check this scope later